### ISSUE TITLE
Loading Overlay doesn't work with spinner/animate

### ISSUE THREAD

**Issue by irtiza-rizvi** [derrickpelletier/react-loading-overlay#21]
I am using React 16.2 with Redux 3.7.2.  I get an error using this component but only when setting `spinner` or `animate` to true.

Here is the basic structure of my component:

```
import React from 'react';
import Loadable from 'react-loading-overlay';

class MyComponent extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
    	<div>
	    	<Loadable
			active={this.props.loading}
			spinner
			text="Loading..."
		>
			<MySubComponent .../>
	    	</Loadable>
		...
	</div>
    }
}
```

Here is the error:

```
app.bundle.js:14462 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `LoadingOverlayWrapper`.
    in LoadingOverlayWrapper (created by MyComponent)
    in div (created by MyComponent)
    ...

Uncaught Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

My guess is `FirstChild` is returning null/undefined in the `CSSTransitionGroup` here: https://github.com/derrickpelletier/react-loading-overlay/blob/master/src/LoadingOverlay.js#L39

Please advise.

**Comment by kika-kaki** [comment]
Any progress on this? I have the same issue

**Comment by derrickpelletier** [comment]
This is resolved i believe, sorry for the delay.
It's fixed in v1.0.0.
